/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:45356'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:44310'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:41402'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:46321'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:38277'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:42033'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:45124'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:37052'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:34702'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:36417'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:40561'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:33218'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:38758'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:40570'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:45307'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:40892'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:45158'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:42935'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:37515'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:45652'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:35106'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:33558'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:44999'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:44532'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:33932'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:46646'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:35934'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.10.111:35372'
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:37369
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:37369
distributed.worker - INFO -              nanny at:        172.30.10.111:46321
distributed.worker - INFO -              bokeh at:        172.30.10.111:42139
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-zrd5tkh7
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:36172
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:36172
distributed.worker - INFO -              nanny at:        172.30.10.111:45652
distributed.worker - INFO -              bokeh at:        172.30.10.111:36709
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-rnnu9n4c
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:37193
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:37193
distributed.worker - INFO -              nanny at:        172.30.10.111:35106
distributed.worker - INFO -              bokeh at:        172.30.10.111:37911
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-7k156ag3
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:42216
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:42216
distributed.worker - INFO -              nanny at:        172.30.10.111:35372
distributed.worker - INFO -              bokeh at:        172.30.10.111:37899
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-yh4yonk0
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:36926
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:36926
distributed.worker - INFO -              nanny at:        172.30.10.111:42935
distributed.worker - INFO -              bokeh at:        172.30.10.111:35919
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-79x68smc
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:38025
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:38025
distributed.worker - INFO -              nanny at:        172.30.10.111:40892
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:36911
distributed.worker - INFO -              bokeh at:        172.30.10.111:43617
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:36911
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO -              nanny at:        172.30.10.111:45307
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -              bokeh at:        172.30.10.111:46540
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-99l7dpks
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-pe6ibek7
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:45058
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:45058
distributed.worker - INFO -              nanny at:        172.30.10.111:40561
distributed.worker - INFO -              bokeh at:        172.30.10.111:42392
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-qc90g6mg
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:37201
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:37201
distributed.worker - INFO -              nanny at:        172.30.10.111:44532
distributed.worker - INFO -              bokeh at:        172.30.10.111:36632
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-zzeqtb_e
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:46159
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:46159
distributed.worker - INFO -              nanny at:        172.30.10.111:44310
distributed.worker - INFO -              bokeh at:        172.30.10.111:40868
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-xc2zpswd
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:46727
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:46727
distributed.worker - INFO -              nanny at:        172.30.10.111:41402
distributed.worker - INFO -              bokeh at:        172.30.10.111:36004
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-kfz_ha26
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:41481
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:41481
distributed.worker - INFO -              nanny at:        172.30.10.111:33218
distributed.worker - INFO -              bokeh at:        172.30.10.111:40712
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-tulmtjo6
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:41202
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:41202
distributed.worker - INFO -              nanny at:        172.30.10.111:34702
distributed.worker - INFO -              bokeh at:        172.30.10.111:44299
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-7oj29nci
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:44159
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:44159
distributed.worker - INFO -              nanny at:        172.30.10.111:36417
distributed.worker - INFO -              bokeh at:        172.30.10.111:42446
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-b_6ei38w
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:36028
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:36028
distributed.worker - INFO -              nanny at:        172.30.10.111:38277
distributed.worker - INFO -              bokeh at:        172.30.10.111:40674
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-gj6gi5vw
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:39072
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:39072
distributed.worker - INFO -              nanny at:        172.30.10.111:38758
distributed.worker - INFO -              bokeh at:        172.30.10.111:45330
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-zy0f1dp4
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:37920
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:37920
distributed.worker - INFO -              nanny at:        172.30.10.111:37515
distributed.worker - INFO -              bokeh at:        172.30.10.111:32836
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-ku5cszoz
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:46728
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:46728
distributed.worker - INFO -              nanny at:        172.30.10.111:37052
distributed.worker - INFO -              bokeh at:        172.30.10.111:42299
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-5ys1ilii
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:39840
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:39840
distributed.worker - INFO -              nanny at:        172.30.10.111:45124
distributed.worker - INFO -              bokeh at:        172.30.10.111:33450
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-yaxs8cpp
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:34388
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:34388
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -              nanny at:        172.30.10.111:42033
distributed.worker - INFO -              bokeh at:        172.30.10.111:43276
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-z0reb6nf
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:44743
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:44743
distributed.core - INFO - Starting established connection
distributed.worker - INFO -              nanny at:        172.30.10.111:45356
distributed.worker - INFO -              bokeh at:        172.30.10.111:39283
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-ns6xudmc
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:36846
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:36846
distributed.worker - INFO -              nanny at:        172.30.10.111:35934
distributed.worker - INFO -              bokeh at:        172.30.10.111:37344
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-xe4ocxo7
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:36209
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:36209
distributed.worker - INFO -              nanny at:        172.30.10.111:46646
distributed.worker - INFO -              bokeh at:        172.30.10.111:39441
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-j_1g_tsj
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:45550
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:45550
distributed.worker - INFO -              nanny at:        172.30.10.111:45158
distributed.worker - INFO -              bokeh at:        172.30.10.111:43118
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-z743bu06
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:34481
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:34481
distributed.worker - INFO -              nanny at:        172.30.10.111:44999
distributed.worker - INFO -              bokeh at:        172.30.10.111:41412
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-xxpsv4fi
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:35821
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:35821
distributed.worker - INFO -              nanny at:        172.30.10.111:40570
distributed.worker - INFO -              bokeh at:        172.30.10.111:36285
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-5i78ysoh
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:44145
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:44145
distributed.worker - INFO -              nanny at:        172.30.10.111:33558
distributed.worker - INFO -              bokeh at:        172.30.10.111:41025
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-pvujpus2
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:44472
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:44472
distributed.worker - INFO -              nanny at:        172.30.10.111:33932
distributed.worker - INFO -              bokeh at:        172.30.10.111:44601
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-iaieto5g
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - ERROR - Worker stream died during communication: tcp://172.30.10.111:44145
Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/tcp.py", line 194, in read
    n_frames = yield stream.read_bytes(8)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 1827, in gather_dep
    self.rpc, deps, worker, who=self.address
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 2975, in get_data_from_worker
    max_connections=max_connections,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 533, in send_recv
    response = yield comm.read(deserializers=deserializers)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/tcp.py", line 214, in read
    convert_stream_closed_error(self, e)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/tcp.py", line 139, in convert_stream_closed_error
    raise CommClosedError("in %s: %s: %s" % (obj, exc.__class__.__name__, exc))
distributed.comm.core.CommClosedError: in <closed TCP>: ConnectionResetError: [Errno 104] Connection reset by peer
distributed.nanny - WARNING - Worker process 3798 was killed by signal 11
distributed.worker - ERROR - Worker stream died during communication: tcp://172.30.10.111:44145
Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/tcp.py", line 194, in read
    n_frames = yield stream.read_bytes(8)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 1827, in gather_dep
    self.rpc, deps, worker, who=self.address
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 2975, in get_data_from_worker
    max_connections=max_connections,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 533, in send_recv
    response = yield comm.read(deserializers=deserializers)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/tcp.py", line 214, in read
    convert_stream_closed_error(self, e)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/tcp.py", line 139, in convert_stream_closed_error
    raise CommClosedError("in %s: %s: %s" % (obj, exc.__class__.__name__, exc))
distributed.comm.core.CommClosedError: in <closed TCP>: ConnectionResetError: [Errno 104] Connection reset by peer
distributed.worker - ERROR - Worker stream died during communication: tcp://172.30.10.111:44145
Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/tcp.py", line 194, in read
    n_frames = yield stream.read_bytes(8)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 1827, in gather_dep
    self.rpc, deps, worker, who=self.address
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 2975, in get_data_from_worker
    max_connections=max_connections,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 533, in send_recv
    response = yield comm.read(deserializers=deserializers)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/tcp.py", line 214, in read
    convert_stream_closed_error(self, e)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/tcp.py", line 141, in convert_stream_closed_error
    raise CommClosedError("in %s: %s" % (obj, exc))
distributed.comm.core.CommClosedError: in <closed TCP>: Stream is closed
distributed.nanny - WARNING - Restarting worker
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:38706
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:38706
distributed.worker - INFO -              nanny at:        172.30.10.111:33558
distributed.worker - INFO -              bokeh at:        172.30.10.111:42117
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-16kj0v0z
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - ERROR - Worker stream died during communication: tcp://172.30.10.111:44145
Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/tcp.py", line 359, in connect
    ip, port, max_buffer_size=MAX_BUFFER_SIZE, **kwargs
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/tcpclient.py", line 280, in connect
    af, addr, stream = await connector.start(connect_timeout=timeout)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/tcpclient.py", line 143, in on_connect_done
    stream = future.result()
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 218, in connect
    quiet_exceptions=EnvironmentError,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/tcp.py", line 371, in connect
    convert_stream_closed_error(self, e)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/tcp.py", line 139, in convert_stream_closed_error
    raise CommClosedError("in %s: %s: %s" % (obj, exc.__class__.__name__, exc))
distributed.comm.core.CommClosedError: in <distributed.comm.tcp.TCPConnector object at 0x2b7f7d698c50>: ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 1827, in gather_dep
    self.rpc, deps, worker, who=self.address
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 2965, in get_data_from_worker
    comm = yield rpc.connect(worker)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 864, in connect
    connection_args=self.connection_args,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 228, in connect
    _raise(error)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 207, in _raise
    raise IOError(msg)
OSError: Timed out trying to connect to 'tcp://172.30.10.111:44145' after 10 s: in <distributed.comm.tcp.TCPConnector object at 0x2b7f7d698c50>: ConnectionRefusedError: [Errno 111] Connection refused
distributed.worker - ERROR - Worker stream died during communication: tcp://172.30.10.111:44145
Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 218, in connect
    quiet_exceptions=EnvironmentError,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
tornado.util.TimeoutError: Timeout

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 1827, in gather_dep
    self.rpc, deps, worker, who=self.address
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 2965, in get_data_from_worker
    comm = yield rpc.connect(worker)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 864, in connect
    connection_args=self.connection_args,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 230, in connect
    _raise(error)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 207, in _raise
    raise IOError(msg)
OSError: Timed out trying to connect to 'tcp://172.30.10.111:44145' after 10 s: in <distributed.comm.tcp.TCPConnector object at 0x2b7f7d5a6860>: ConnectionRefusedError: [Errno 111] Connection refused
distributed.nanny - WARNING - Worker process 3794 was killed by signal 11
distributed.nanny - WARNING - Restarting worker
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:45303
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:45303
distributed.worker - INFO -              nanny at:        172.30.10.111:45652
distributed.worker - INFO -              bokeh at:        172.30.10.111:33839
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-5qzlni1i
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.nanny - WARNING - Worker process 3768 was killed by signal 11
distributed.nanny - WARNING - Restarting worker
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:  tcp://172.30.10.111:39863
distributed.worker - INFO -          Listening to:  tcp://172.30.10.111:39863
distributed.worker - INFO -              nanny at:        172.30.10.111:45124
distributed.worker - INFO -              bokeh at:        172.30.10.111:42027
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-plj5w1ip
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:40598
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.utils_perf - INFO - full garbage collection released 39.30 MB from 1112 reference cycles (threshold: 10.00 MB)
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - INFO - -------------------------------------------------
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - -------------------------------------------------
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - INFO - -------------------------------------------------
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.worker - WARNING - Heartbeat to scheduler failed
distributed.utils_perf - INFO - full garbage collection released 14.54 MB from 1342 reference cycles (threshold: 10.00 MB)
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:40598
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:34481
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:36911
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:41481
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:45550
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:42216
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:36028
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:35821
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:44472
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:44999'
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:37369
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:33218'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:45307'
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:36926
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:46727
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:44743
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:39072
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:36209
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:38706
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:39863
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:36846
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:38025
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:45058
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:37920
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:46728
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:34388
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:37193
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:40570'
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:44159
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:37201
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:41202
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:46159
distributed.worker - INFO - Stopping worker at tcp://172.30.10.111:45303
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:33932'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:45158'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:46321'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:35372'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:42935'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:42033'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:35106'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:41402'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:35934'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:38758'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:33558'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:40561'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:37052'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:46646'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:40892'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:45356'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:37515'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:44532'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:36417'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:44310'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:34702'
distributed.nanny - INFO - Closing Nanny at 'tcp://172.30.10.111:38277'
distributed.dask_worker - INFO - End worker
distributed.process - WARNING - reaping stray process <ForkServerProcess(ForkServerProcess-30, started daemon)>
distributed.process - WARNING - reaping stray process <ForkServerProcess(ForkServerProcess-31, started daemon)>
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/dask/config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
